experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -69,6 +73,15 @@ type EvaluateWaitError struct {
 	msg string
 }
 
+const (
+	finalizerName    = "compositions.google.com/expander-finalizer"
+	stagesAnnotation = "compositions.google.com/expander-stages"
+	nsAnnotation     = "applyset.kubernetes.io/additional-namespaces"
```

Comment:
Do you think we can use this instead of defining it here ? 

https://pkg.go.dev/sigs.k8s.io/kubebuilder-declarative-pattern/applylib@v0.0.0-20240813003123-2a9fe229ddb2/forked/github.com/kubernetes/kubectl/pkg/cmd/apply#pkg-constants

Reply:
Yes, good call.